notes/interview-summary.vbs

```diff
@@ -583,11 +573,11 @@ function define_main_dialog()
 			Text 330, 135, 35, 10, "Hispanic?"
 			Text 330, 160, 50, 10, "Race"
 			Text 70, 200, 145, 10, "Which programs is this person requesting?"
-			Text 70, 255, 55, 10, "Intends to reside in MN"
+			Text 70, 255, 70, 10, "Intends to reside in MN"
```

Comment:
Text field for "Intends to reside in MN" is cut off - looks like it needs to be at least 80 wide